pyqtgraph/graphicsItems/ScatterPlotItem.py

```diff
@@ -103,138 +120,191 @@ def makeSymbolPixmap(size, pen, brush, symbol):
     img = renderSymbol(symbol, size, pen, brush)
     return QtGui.QPixmap(img)
 
+
+def _mkPen(*args, **kwargs):
```

Comment:
I know you explain in the comment section above about re-using pens for performance reasons; but there should be doc-strings here for these helpers methods explaining why they're here.